stable/prometheus/templates/server-storageclass.yaml

```diff
@@ -0,0 +1,16 @@
+kind: StorageClass
```

Comment:
The StorageClass should only be created if the persistentVolume specifies a storageClass

Reply:
done